client/auth/native-login/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import LoginBlock from 'blocks/login';
+import { localize } from 'i18n-calypso';
+
+const Login = ( { translate } ) => (
+	<Main className="native-login">
+		<div>
+			<div className="native-login__header">
+				<Gridicon icon="user-circle" size={ 72 } />
+				<div>{ translate( 'You are signed out' ) }</div>
+			</div>
+			<div className="native-login__container">
+				<LoginBlock
+					title={ translate( 'Sign in to connect to WordPress.com' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Sign in to WordPress.com' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).